two-factor.php

```diff
@@ -155,7 +158,10 @@ function() use ( $is_user_using_two_factor ) {
 	}
 }
 
-add_action( 'muplugins_loaded', 'wpcom_enable_two_factor_plugin' );
+// Do not enable 2FA plugin if WordPress is executed in the context of the CLI
```

Comment:
I think instead of wrapping it here maybe just move that inside the `wpcom_enable_two_factor_plugin`

Reply:
Ok I will try that and see how that works.